include/mbedtls/ecp.h

```diff
@@ -53,139 +65,150 @@ extern "C" {
 #endif
 
 /**
- * Domain parameters (curve, subgroup and generator) identifiers.
+ * Definition of domain parameter identifiers: curve, subgroup and generator.
  *
- * Only curves over prime fields are supported.
+ * \note Only curves over prime fields are supported.
  *
  * \warning This library does not support validation of arbitrary domain
- * parameters. Therefore, only well-known domain parameters from trusted
+ * parameters. Therefore, only standardized domain parameters from trusted
```

Comment:
waiting for a decision on this...

Reply:
It's not a blocker for me, but I still like the combination of `standardized` and `trusted sources` (which rules out dubious stuff like RFC 5114) more than `well-known`. @gilles-peskine-arm Your call in the end.

Reply:
I was mostly thinking of Curve25519, which for a long time was only a DJB publication and not an official standard, but it's in an RFC and a NIST standard now. Anyway, I prefer “well-known” but it isn't a blocker, so please leave “standardized” unless @yanesca doesn't like it either.

Reply:
I am happy with any of these suggestions, because I don't see how being slightly vague here can lead to user errors.

(If I had to phrase this I would go for `domain parameters for secure named curves`, because it does not restrict us to standardised curves and gives us the liberty of remove domain parameters not only when they are not generated in a trusted way, but even when we don't think that they are cryptographically secure anymore)

Reply:
Guys, can you please advise what the final decision on this is?

Reply:
Since there are no strong feelings, let's keep this simple: please keep the current wording.

Reply:
ok. Keeping the current as "standardized"